sys/include/phydat.h

```diff
@@ -180,38 +180,26 @@ const char *phydat_unit_to_str(uint8_t unit);
 char phydat_prefix_from_scale(int8_t scale);
 
 /**
- * @brief   Scale an integer value to fit into a @ref phydat_t
+ * @brief   Scale integer value(s) to fit into a @ref phydat_t
  *
- * Insert @p value at position @p index in the given @p dat while rescaling all
- * numbers in @p dat->val so that @p value fits inside the limits of the data
- * type, [@ref PHYDAT_MIN, @ref PHYDAT_MAX], and update the stored scale factor.
- * The result will be rounded towards zero (the standard C99 integer division
- * behaviour).
- * The final parameter @p prescale can be used to chain multiple calls to
- * this function in order to fit multidimensional values into the same phydat_t.
- *
- * The code example below shows how to chain multiple calls via the @p prescale parameter
+ * Inserts the @p values in the given @p dat so that all @p dim values in
+ * @p values fit inside the limits of the data type,
+ * [@ref PHYDAT_MIN, @ref PHYDAT_MAX], and updates the stored scale factor.
+ * The value is rounded towards infinity, e.g. `0.5` and `0.6` are rounded to
+ * `1`, `0.4` and `-0.4` are rounded to `0`, `-0.5` and `-0.6` are rounded to
+ * `-1`.
```

Comment:
Towards infinity (+infinity) means that 0.1 would be rounded to 1, -0.9 would be rounded to 0. This is rounding towards nearest integer (round half away from zero).

Reply:
I see your point, this name is indeed misleading. [Wikipedia refers to it as round half away from zero](https://en.wikipedia.org/wiki/Rounding#Round_half_away_from_zero) in addition to "towards infinity". Typing it into some internet search engine also shows that "rounding away from zero" is the more widely used term then "towards infinity".